packages/react-examples/src/react/Nav/Nav.FabricDemoApp.Example.tsx

```diff
@@ -74,6 +74,13 @@ const navLinkGroups: INavLinkGroup[] = [
 
 export const NavFabricDemoAppExample: React.FunctionComponent = () => {
   return (
-    <Nav styles={navStyles} ariaLabel="Nav example similar to one found in this demo page" groups={navLinkGroups} />
+    <Nav
+      styles={navStyles}
+      ariaLabel="Nav example similar to one found in this demo page"
+      groups={navLinkGroups}
+      focusZoneProps={{
+        defaultTabbableElement: "a[title='MarqueeSelection']",
```

Comment:
Should make a separate example showcasing overriding the default tabbable element.